.github/workflows/produce_data.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         run: |
           python3 infra/src/generate_data.py --run_id ${{ steps.get-run-id-and-attempt.outputs.run-id }}
           # Workaround: Copy file to avoid GH upload filename limitations
-          cp pipeline_${{ steps.get-run-id-and-attempt.outputs.run-id }}*.json pipeline_${{ steps.get-run-id-and-attempt.outputs.run-id }}.json
```

Comment:
should this be a `mv`, if we only want to rename the files?

Reply:
No, we still need to upload the file to SFTP with original name.   

The script will generate file named according to data team spec 
```pipeline_11345421170_2024-10-15T11:50:00.json```  and we are uploading that one to SFTP.
We also want to upload the same file to artifacts, but Github doesn't allow filenames with colon ":", so we create a copy of the same file named 
```pipelinecopy_11345421170.json``` and upload that one to artifacts.